lib/cucumber/formatter/message_builder.rb

```diff
@@ -192,10 +193,13 @@ def on_test_step_finished(event)
 
         result_message = result.to_message
         if result.failed? || result.pending?
+          message_element = result.failed? ? result.exception : result
```

Comment:
@ehuelsmann / @davidjgoss - This is where I'm tripping up now and cannot explain it.

In the CCK from here (https://github.com/cucumber/compatibility-kit/commit/a3546f798a0caa09715e82027608e93b83f6d0a4#diff-7df09050f374e2ce8fc33ec3c05df9b40844714d7d622da7b1a56a73fd7f10bd) I can see for over 2 years we've never had a message property in a `PENDING` message. 


However..... We've had little coverage of PENDING messages - just that one scenario.

From the above you can see previously we were always treating pending steps here as requiring 3 properties (Correct on old schema - `status`, `duration` and `message`). This PR adds the fourth one for backwards compatibility.

However it seems as though something went awry in the CCK reconciliation updates I've been doing. But I cannot spot what. It seems as though the CCK **never** had this `message` property set (Last 2 years).

I probably need to explain it better. But not 100% sure where I've gone wrong / what's wrong